src/config/Blocks/listingVariations.js

```diff
@@ -151,7 +151,7 @@ const italiaListingVariations = [
     cloneData: cloneBlock,
   },
   {
-    id: 'quaresImageTemplate',
+    id: 'squaresImageTemplate',
```

Comment:
Sarebbe bellissimo risolvere questo typo ma ho paura che crei problemi. Se non vado errato, andrebbe a "rompere" tutti quei blocchi che hanno già questa variazione selezionata, perché andrebbero a cercare nella config la configurazione che ha l'id vecchio. Se mi mostri che sbaglio mi sta benissimo cambiarlo :D
@giuliaghisini ti torna?

Reply:
mi era sfuggito..  mi torna mi torna. non possiamo cambiarlo altrimenti non si vedono piu i blocchi vecchi fatti con il nome sbagliato :-(

Reply:
Non ti sbagli, è così. Però, il fallback è una card semplice. Quindi non va in errore né si rompe in modo brutto, ma viene messa la variante con la card "classica". 
Se lo consideriamo breaking e vogliamo evitare, posso ripristinare il typo

Reply:
Sì è breaking allora, evitiamolo.

Reply:
Ok, ripristinato